src/patterns/gen-ai/aws-model-deployment-sagemaker/jumpstart-sagemaker-endpoint.ts

```diff
@@ -183,6 +183,10 @@ export class JumpStartSageMakerEndpoint extends SageMakerEndpointBase {
       ...this.environment,
     };
 
+    if (environment.SAGEMAKER_SUBMIT_DIRECTORY) {
+      delete environment.SAGEMAKER_SUBMIT_DIRECTORY;
```

Comment:
any reason to delete this variable ?

Reply:
based on my understanding this env var is for model training.
and somehow, when we have this env var, the Jumpstart UI console complaints that it is not detected as jumpstart model, hence the option to `Test Inference` is disabled.
after I removed this env var, now I can do manual test via `Test Inference` via console.